src/ray/gcs/callback.h

```diff
@@ -53,6 +53,14 @@ using SubscribeCallback = std::function<void(const ID &id, const Data &result)>;
 template <typename Data>
 using ItemCallback = std::function<void(const Data &result)>;
 
+/// This callback is used to receive scan result from storage.
```

Comment:
```suggestion
/// This callback is used to receive a large amount of results.
/// It will be called one or multiple times, with one small chunk of results each time, until `has_more` is false. 
```
`scan` seems to be unclear and specific to redis. What about naming it `ChunkedCallback`.

Reply:
done